### PR TITLE
feat: migrate mobile learn test to playwright

### PR DESCRIPTION
**Pull request by Nirajn2311**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

I created another playwright config as it felt easier to manage the different browsers and settings. Also note that the mobile workflow will run on dispatch only so it needs to be run once after this merged.

### CODE REVIEW

**Review by ojeytonwilliams — COMMENTED** [review]
Can we delete the Cypress tests while we're at it?

**Comment by Nirajn2311** [comment]
> Can we delete the Cypress tests while we're at it?

Nice catch, I forget to remove it after migrating

**Review by ojeytonwilliams — APPROVED** [review]
Cheers Niraj. LGTM 👍

File: playwright-mobile.config.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    /* ====================================================== */
+    /* Test against branded browsers. */
+    // {
+    //   name: 'Microsoft Edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' }
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' }
+    // }
```
Comment (naomi-lgbt) [inline playwright-mobile.config.ts:55]:
```suggestion
```

**Review by naomi-lgbt — COMMENTED** [review]
Non-blocking nit: 

We should not need to run the mobile tests on desktop browsers, so there's no sense in keeping them commented out in the config

File: playwright-mobile.config.ts (hunk 1 above)
Comment (Nirajn2311) [reply playwright-mobile.config.ts:55]: Good point. I'll remove them.

**Review by naomi-lgbt — APPROVED** [review]

**Comment by Nirajn2311** [comment]
Perfect, the action works as expected with even the one test that we know will fail. Not bad for a first try 🙂

**Pull request merged**